### PR TITLE
impr(UTAPI-83): Add warp 10 release workflow

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: .github/workflows/release-warp10.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  release:
+    if: ${{ inputs.create-github-release }}
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: Release scality/warp10:${{ github.event.inputs.tag }}
```
Comment (jonathan-gramain) [inline .github/workflows/release-warp10.yaml]: This looks more accurate but please correct my suggestion if it is not correct:

```suggestion
          name: Release registry.scality.com/utapi/warp10:${{ github.event.inputs.tag }}-warp10
```

**Review by jonathan-gramain — APPROVED** [review]

**Review by Emylou-s — CHANGES_REQUESTED** [review]
There seems to be significant changes between the dockerfile used in utapi repo and the one used in federation.
Among others one I can see is the line in Federation : 

`# Install protobuf extestion
ADD ./warp10-ext-protobuf-1.1.0-uberjar.jar /opt/warp10/lib/
`

How can we make sure the generated image will be compatible with what's built in Federation.

I have the feeling that there will be significant work needed in Federation to match this new image.

**Comment by tmacro** [comment]
The image is compatible with the one in federation. The only difference aside from the bump to the protobuf extension is the
inclusion of the sensision prometheus exporter which is disabled by default.

**Review by Emylou-s — APPROVED** [review]

**Pull request merged**